sources/ghostdog/src/main.rs

```diff
@@ -48,6 +53,40 @@ struct EbsDeviceNameArgs {
     device: PathBuf,
 }
 
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "match-nvidia-driver")]
+/// Returns if devices on the PCI bus support the provided driver.
+struct MatchNvidiaDriverArgs {
+    #[argh(positional)]
+    driver_name: String,
+}
+
+#[derive(Deserialize)]
+/// Open GPU struct for comparing PCI ID's to a known list of supported devices.
+enum SupportedDevicesConfiguration {
+    #[serde(rename = "open-gpu")]
+    OpenGpu(Vec<GpuDeviceData>),
+}
+
+#[derive(PartialEq, Debug, Deserialize)]
+/// The GPU Device Data contains various features of the device. Only Name, Device ID, and Features are required
+/// a particular device
```

Comment:
nit: missing word, guessing ...?
```suggestion
/// The GPU Device Data contains various features of the device. Only Name, Device ID, and Features are required
/// for a particular device
```